x-pack/plugins/security_solution/server/endpoint/routes/policy/handlers.test.ts

```diff
@@ -49,11 +49,7 @@ describe('test policy response handler', () => {
 
     it('should return the latest policy response for a host', async () => {
       const response = createSearchResponse(new EndpointDocGenerator().generatePolicyResponse());
-      const hostPolicyResponseHandler = getHostPolicyResponseHandler({
-        logFactory: loggingSystemMock.create(),
-        service: endpointAppContextService,
-        config: () => Promise.resolve(createMockConfig()),
-      });
+      const hostPolicyResponseHandler = getHostPolicyResponseHandler();
```

Comment:
ok :+1: . But why was this part of this PR?

Reply:
Unused argument? I guess I got too much into cleanup mode 😅